app/src/main/res/values/strings.xml

```diff
@@ -232,6 +232,11 @@
     <string name="first_run_upgrade_page_text2">2. Access your favorite sites quickly by %s</string>
     <string name="first_run_upgrade_page_text2_highlight">adding them to the home screen!</string>
 
+    <!--Update tour for pin sites to home page-->
+    <string name="second_run_upgrade_page_title">Pin your favorite sites</string>
```

Comment:
Both `second_run_upgrade_page_title` and `second_run_upgrade_page_text` are removed by @proevan. He mentioned that there is no pin site onboarding anymore. Could you help to confirm?